airbyte-integrations/connectors/source-twilio/acceptance-test-config.yml

```diff
@@ -46,6 +46,8 @@ acceptance_tests:
             bypass_reason: "very volatile data"
           - name: "usage_records"
             bypass_reason: "very volatile data"
+          - name: "step"
+            bypass_reason: "stream not filled yet"
```

Comment:
@sajarin for GA connectors we must add data in the new stream. Only if there is a very specific condition where isn't possible we can skip them. Please take a look in the Twilio API and check how you can create some data points in the step.